File: internal/flowdoc/testdata/verify-symbols/sample.py
"""Python fixture for verify symbol-grammar tests."""

MAX_FRAMES = 128

class CanMonitor:
    def __init__(self, name):
        self.name = name

    def start(self):
        return self.name


def load_config(path):
    return {"path": path}


async def run_replay_mode(file):
    return None

File: internal/flowdoc/testdata/verify-symbols/sample.ts
// TypeScript fixture for verify symbol-grammar tests.

export const MAX_FRAMES = 128;

export interface FrameLike {
    id: number;
}

export class CanMonitor {
    constructor(public name: string) {}
}

export async function runReplayMode(file: string): Promise<void> {
    return;
}

function loadConfig(path: string): object {
    return { path };
}
